test/parallel/test-runner-root-after-with-refed-handles.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const common = require('../common');
+const { before, after, test } = require('node:test');
+const { createServer } = require('node:http');
+
+let server;
+
+before(common.mustCall(() => {
+  server = createServer();
+
+  return new Promise(common.mustCall((resolve, reject) => {
+    server.listen(0, common.mustCall((err) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve();
+      }
+    }));
+  }));
+}));
+
+after(common.mustCall(() => {
+  server.close();
```

Comment:
@cjihrig would it be worth ensuring the after hook also waits for cb?
```suggestion
  server.close(common.mustCall());
```

Reply:
That's not a bad idea. Honestly though, the CI was a pain to get passing yesterday and I would prefer not to start over for that change.